install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -85,6 +85,30 @@ data:
     outboundTrafficPolicy:
       mode: {{ .Values.global.outboundTrafficPolicy.mode }}
 
+    {{- if .Values.global.configRootNamespace }}
+    # The namespace to treat as the administrative root namespace for istio
+    # configuration. Set this field to a dedicated namespace if you want to
+    # all sidecars to be able to communicate with services in their
+    # namespace alone. This dedicated namespace should have a default
+    # Sidecar config
+    rootNamespace: {{ .Values.global.configRootNamespace }}
+    {{- end }}
+
+    {{- if .Values.global.defaultConfigVisibilitySettings }}
```

Comment:
I'm more and more tempted to just have a section under global, named 'mesh' and use exactly the format 
from this config map, and import it directly - instead of having 2 different sets of keywords and out of sync naming.

Not an issue for this PR, but in general.